crates/proc-macro-srv/proc-macro-test/build.rs

```diff
@@ -17,7 +17,6 @@ use cargo_metadata::Message;
 
 fn main() {
     println!("cargo:rerun-if-changed=imp");
-    println!("cargo:rerun-if-env-changed=PROC_MACRO_TEST_TOOLCHAIN");
```

Comment:
Don't we want this?

Reply:
do we? I checked our CI and bootstrap in rustc, neither of which use this.

Reply:
The build script uses it, so it will catch whoever tries to change it (you?) off-guard.

Reply:
Well no, I removed it from the build script altogether

Reply:
Oh, sorry, I misread the diff.